database/migrations/2021_03_09_212637_create_comments_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\BookingRequest;
+use App\Models\User;
+
+class CreateCommentsTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('comments', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->foreignIdFor(BookingRequest::class, 'booking_id');
+            $table->foreignIdFor(User::class)->nullable();
```

Comment:
Why is this nullable? For system comments?

Reply:
yes